### PR TITLE
Drive metadata without using the APIs

### PR DESCRIPTION
**Pull request by samiyaakhtar**

### CODE REVIEW

File: client/constants.py [hunk 1]
```diff
@@ -23,10 +23,12 @@ class UrlTypes(object):
     GENERAL = "general"
     WIKI = "wikipedia"
     YOUTUBE = "youtube"
+    DOCS = "google docs"
 
     special_urls = {
         WIKI: "wikipedia.org",
-        YOUTUBE: "youtube.com"
+        YOUTUBE: "youtube.com",
+        DRIVE: "docs.google.com"
```
Comment (MeenuV) [inline client/constants.py]: Let's keep it consistentent to either `DOCS` or `DRIVE`. I would suggest `DOCS`

File: client/url_utils.py [hunk 2]
```diff
@@ -38,5 +38,7 @@ def get_url_type(url):
         return UrlTypes.WIKI
     elif UrlTypes.get_special_url(UrlTypes.YOUTUBE) in url:
         return UrlTypes.YOUTUBE
+    elif UrlTypes.get_special_url(UrlTypes.DOCS) in url:
+        return UrlTypes.DRIVE
```
Comment (MeenuV) [inline client/url_utils.py]: You need to change to docs here as well

File: client/scraper/metadata.py [hunk 3]
```diff
@@ -19,7 +23,55 @@ def __init__(self, response):
...
+            media_list[FieldKeyword.COUNT] = 1
+            media_list[FieldKeyword.DATA] = []
+            data = {
+                FieldKeyword.TYPE: FieldValue.IMAGE,
+                FieldKeyword.SRC: image_url
+            }
+            media_list[FieldKeyword.DATA].append(data)
+            self.prop_map[FieldKeyword.MEDIA] = media_list
+
     def to_json(self):
```
Comment (MeenuV) [inline client/scraper/metadata.py]: Can you move this to the end of this class?

Comment (samiyaakhtar) [inline client/scraper/metadata.py]: ok

**Comment by samiyaakhtar** [comment]
@AliceYuan: r+

**Pull request merged**